documentation/API_documentation/qod-provisioning-API-Readiness-Checklist.md

```diff
@@ -9,10 +9,10 @@ Checklist for qod-provisioning v0.1.0-rc.1 in r1.1
 |  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   | Y    |      |
 |  4 | API versioning convention applied            |   M   |         M         |    M    |    M   | Y    |      |
 |  5 | API documentation                            |   M   |         M         |    M    |    M   | Y    | inline in YAML |
-|  6 | User stories                                 |   O   |         O         |    O    |    M   | N/A    | |

```

Comment:
Yo can remove the PR: from comments in line 7, as PR should be merged before this one.

Reply:
Done in https://github.com/camaraproject/QualityOnDemand/pull/336/commits/2644cecbb968549d7ffae3531a8386aa3653c307